python/ray/tune/ray_trial_executor.py

```diff
@@ -160,7 +160,8 @@ def __init__(self,
 
         self._avail_resources = Resources(cpu=0, gpu=0)
         self._committed_resources = Resources(cpu=0, gpu=0)
-        self._pg_manager = PlacementGroupManager()
+        self._pg_manager = PlacementGroupManager(
+            prefix=os.getenv("TUNE_PLACEMENT_GROUP_PREFIX", "__tune__"))
```

Comment:
should we add a hex to differentiate among different Tune runs?

Reply:
This is just the prefix here - and it's used to identify leftover placement groups to remove _before_ starting a new Tune run. It thus has to be constant across runs, otherwise removal wouldn't make sense. The trial PGs are actually using unique hex identifiers.

I guess one possibility would be to create a hex, store it in a global variable, and re-use this for sequential runs. Effectively this would mean the auto-removal process will only be triggered for sequential runs (such as in our tests). Parallel trials in different remote functions would work out of the box. Parallel trials using shared global state (threads?) would still interfere, but they do this right now, too.

Hm, this might be a good idea, I'll think about it a bit more.

Reply:
I ended up implementing this - I'll see if the tests pass, but the examples in the issues run for me without problems (other than setting a separate logdir)